Source/missiles.cpp

```diff
@@ -2208,105 +2208,54 @@ BOOLEAN __fastcall CheckIfTrig(int x, int y)
 
 void __fastcall AddTown(int mi, int sx, int sy, int dx, int dy, int midir, int mienemy, int id, int dam)
 {
-	int v9;    // ebx
-	int v10;   // esi
-	int v11;   // edi
-	int v12;   // eax
-	int v13;   // ecx
-	char *v14; // eax
-	int v15;   // eax
-	//int v16; // eax
-	int v17;         // ecx
-	int v18;         // eax
-	int v19;         // eax
-	int v20;         // ecx
-	int v21;         // eax
-	int v22;         // ST0C_4
-	int CrawlNum[6]; // [esp+Ch] [ebp-28h]
-	int i;           // [esp+24h] [ebp-10h]
-	char *v25;       // [esp+28h] [ebp-Ch]
-	int v26;         // [esp+2Ch] [ebp-8h]
-	int v27;         // [esp+30h] [ebp-4h]
-	int x;           // [esp+40h] [ebp+Ch]
+	int i, j, k, mx, tx, ty, CrawlNum[6] = { 0, 3, 12, 45, 94, 159 };

```

Comment:
```suggestion
	int i, j, k, mx, tx, ty;
	int CrawlNum[6] = { 0, 3, 12, 45, 94, 159 };

```

Reply:
Hmm, I tried to commit the suggestion, but seems I can't:

![2019-03-14-091206_881x275_scrot](https://user-images.githubusercontent.com/1414531/54322631-41a62b00-4639-11e9-9578-c9f4e1896a5e.png)

@qndel. Would you commit it? There should be a button in the GitHub web UI that says *Commit suggestions.

![2019-03-14-091245_221x54_scrot](https://user-images.githubusercontent.com/1414531/54322653-5e426300-4639-11e9-9f67-c44178eb6b24.png)